src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -95,26 +94,14 @@
     return Collections.emptyList();
   }
 
-  /**
-   * Produces the flag form of a file object, using {@link File#toPath()}.
-   *
-   * @return {@code [--name, file.toPath().toString()]} or {@code []} if file is null.
-   */
-  static List<String> filePath(String name, @Nullable File file) {
-    if (file != null) {
-      return path(name, file.toPath());
-    }
-    return Collections.emptyList();
-  }
-
   /**
    * Produces the flag form of a path object, using {@link Path#toString()}.
    *
    * @return {@code [--name, path.toString()]} or {@code []} if path is null or not set.
    */
   static List<String> path(String name, @Nullable Path path) {
     if (path != null && !path.toString().isEmpty()) {
-      return Arrays.asList("--" + name, path.toString());
+      return Arrays.asList("--" + name, path.toAbsolutePath().toString());
```

Comment:
could this cause any behavior changes for the callers or is it the same thing?

Reply:
I looked through it, and it seemed fine. But I'll go through and check again. There actually weren't too many callers.

Reply:
so it looks like this should be okay, the callers (in tests) were all provided with absolute paths. And all commands that execute with this parameter should be taking absolute paths anyway.

Reply:
I've pushed an update to the tests to make it explicit.